discovery-infra/tests/base_test.py

```diff
@@ -258,3 +258,57 @@ def wait_for_nodes_status_installing_or_installed(
                       consts.NodesStatus.INSTALLED],
             interval=30,
         )
+
+    @staticmethod
+    def register_dummy_host(cluster_id, api_client):
+        dummy_host_id = "b164df18-0ff1-4b85-9121-059f10f58f71"
+        api_client.register_host(cluster_id, dummy_host_id)
+
+    @staticmethod
+    def get_cluster_host_ids(cluster_id, api_client):
+        host_ids = []
+        hosts = api_client.get_cluster_hosts(cluster_id)
+
+        for host in hosts:
+            host_ids.append(host['id'])
+        
+        return host_ids
+
+    @staticmethod
+    def host_fail_cluster_install(cluster_id, host_id, api_client):
+        api_client.host_update_progress(cluster_id, host_id, "Failed")
+
+    @staticmethod
+    def complete_cluster_installation(cluster_id, api_client):
+        api_client.complete_cluster_installation(cluster_id=cluster_id, is_success=True)
+
+    @staticmethod
+    def assert_http_error_code(api_client, api_call, status, reason, **kwargs):
+        with pytest.raises(ApiException) as response:
+            api_call(api_client=api_client, **kwargs)
+        assert response.value.status == status
+        assert response.value.reason == reason
+
+    @staticmethod
+    def get_cluster(cluster_id, api_client):
```

Comment:
not sure i understand why we add static function for every api call?

Reply:
I need the functions to be called in the same way for:
https://github.com/openshift/assisted-test-infra/blob/bd4328952346c3cdc17b2ca15f3b3bd3824fb35d/discovery-infra/tests/base_test.py#L286